### DISCUSSION TITLE
Is it possible to use 6 channel image in detectron2 deep learning?

### DISCUSSION THREAD

**Discussion by leesangjoon1** [facebookresearch/detectron2#2885] [General]
I have one 3 channel image (RGB)
and the other 3 channel image (RGB but It is captured using fluorescence light)

So I add and make them 6 channel

But I have a question 

Is it possible to use 6 channel image in detectron2 deep learning?

**Comment by abreufilho** [comment]
I also would like to know how to adapt pretrained weights for multiple channels beyond RGB channels.

**Comment by ppwwyyxx** [comment]
https://github.com/facebookresearch/detectron2/issues/2062 explains how to do this

**Reply by abreufilho** [reply]
Kinda. Actually, i would like to duplicate the weights from a pre-trained model (mask_rcnn_R_50_FPN_3x for instance) for a RGB input image for RGB+RGB (6 channels) image. Is it possible? Or do i have to train the model from scratch anyways?

**Reply by ppwwyyxx** [reply]
you can `pickle.load` the weights and edit it in any ways you want

**Reply by leesangjoon1** [reply]
then is it possible to use translate learning? most of the translate learning weights is fitted to 3 channel weights.

then Don't I adjust the pretrained weights to 6 channel?
